bin/multi-tester

```diff
@@ -43,4 +60,29 @@ if (!getLoader()) {
     exit(1);
 }
 
-exit((new MultiTester())->run($argv) ? 0 : 1);
+try {
+    $multiTester = new MultiTester();
+    $code = $multiTester->run($argv) ? 0 : 1;
+} catch (ZeroProjectsTestedException $e) {
+    showError($e);
```

Comment:
The same way you check `$multiTester->isVerbose()` then you could test `$multiTester->isColored()` so the static method is no longer needed and color could be configurable for errors too.

Reply:
be patient )) optimizations will follow

Reply:
I'll let you know when PR is ready

Reply:
also it seems to me that static method is needed in case when config file is absent